.github/renovate.json5

```diff
@@ -51,12 +51,19 @@
       automerge: false, // just be explicit (false is the default)
     },
 
-    // Reduce awssdk update frequency noise (awssdk has daily releases)
+    // Reduce update frequency for a few dependencies that have a quite high release frequency.
+    //   awssdk (release frequency: daily)
```

Comment:
If we are making renovate only run weekly, do we still need this?

Reply:
Consensus is to keep the current schedule